robot_sort/robot_sort.py

```diff
@@ -97,16 +176,72 @@ def sort(self):
         Sort the robot's list.
```

Comment:
Great job on the last two objectives with "robot_sort.py", @MelvinPeepers! You got 3/3 points + 3/3 points = 6/6 possible points for the "robot_sort()" objectives! 🥇 
<img width="722" alt="Screen Shot 2020-03-14 at 10 21 14 PM" src="https://user-images.githubusercontent.com/16110931/76693987-5f8adf00-6643-11ea-850a-04c3359db9cf.png">

Reply:
<img width="1280" alt="melvin_robot_sort" src="https://user-images.githubusercontent.com/16110931/76693998-6d406480-6643-11ea-815a-2fdbb235ee8b.png">